add_balancer.yml

```
@@ -16,14 +16,20 @@
   pre_tasks:
     - name: Include OS-specific variables
       include_vars: "vars/{{ ansible_os_family }}.yml"
-      when: not ansible_os_family == 'Rocky'
+      when: not ansible_os_family == 'Rocky' and not ansible_os_family == 'AlmaLinux'
 
     # For compatibility with Ansible old versions
     # (support for Rocky Linux has been added to Ansible 2.11)
     - name: Include OS-specific variables
       include_vars: "vars/RedHat.yml"
       when: ansible_os_family == 'Rocky'
 
+    # For compatibility with Ansible old versions
+    # (support for AlmaLinux has been added to Ansible 2.11)
+    - name: Include OS-specific variables
+      include_vars: "vars/RedHat.yml"
+      when: ansible_os_family == 'AlmaLinux'
```

```suggestion
    # For compatibility with Ansible old versions
    # (support for RockyLinux and AlmaLinux has been added to Ansible 2.11)
    - name: Include OS-specific variables
      include_vars: "vars/RedHat.yml"
      when: ansible_os_family == 'Rocky' or ansible_os_family == 'AlmaLinux'
```
reduce the amount of code.